### DISCUSSION TITLE
Chromcast on Smplayer

### DISCUSSION THREAD

**Discussion by Marcosdw** [smplayer-dev/smplayer#86] [General]
Hello,

Would it be possible to have the integration and compatibility of diffusion to chromcast because the current system: https://chromecast.smplayer.info/ is totally obsolete
thanks in advance
Greetings
Marco

Sorry, I use a French / English online translator

**Comment by smplayer-dev** [comment]
Well, if someone knows a library which can work both for windows and linux I'll happily take a look at it.

**Comment by bayvista170** [comment]
This does not work on Linux but it does work on Windows?
